readme.md

```diff
@@ -598,6 +598,17 @@
 
 ## Strategy
 
+### [Ashtapada](https://en.wikipedia.org/wiki/Ashtapada)
+
+> Ashtapada, an ancient game originating in India, is played on an 8x8 grid featuring marked spaces. The game can be played by two to four players, with the objective being to enter and exit both pieces strategically. Players use cowrie shells to determine movement. If a player lands on an opponent's piece, it's sent off the board for re-entry, except on 'crosscut' spaces, where pieces remain safe.
+
+![Ashtapada game image](https://cf.geekdo-images.com/kg1eP5Vdj6LoFqQ2E2Kxew__opengraph/img/lykEixPPGFGql26yrwEYvhj3XMo=/fit-in/1200x630/filters:strip_icc()/pic55539.jpg)
+
+| Players | Min. Age | Time   |
+| ------: | -------: | -----: |
+|   2–4 |          8 | 30–60 Min |
```

Comment:
Where are you getting the stats from? BGG shows 2 players only, neither they nor wikipedia have any age or time frame ranges.

Reply:
Hello,
One of the resources I followed is [Ashtapada](https://a4gamescompany.wordpress.com/ashtapada/). Here, it was mentioned regarding the number of players. When talking about age and time, it's just an estimate I have gathered from a few other resources.
What should I write if I don't have a proper estimate for age and time?

Thanks for your time and consideration.

Reply:
Thank you. I'm considering how best to move forward. It sounds like it's an ancient enough game to not have a lot in the way of documentation 😆